app/controllers/miq_ae_tools_controller.rb

```diff
@@ -452,5 +453,13 @@ def set_session_data
     session[:resolve_tools] = @resolve if @resolve
   end
 
+  def breadcrumbs_options
+    breadcrumbs = [{:title => _("Automation")}, {:title => _("Automate")}]
+    breadcrumbs.push(:title => _("Simulation")) if action_name == "resolve"
+    {
+      :breadcrumbs => breadcrumbs,
+    }
```

Comment:
What's your opinion about:
```ruby
{
  :breadcrumbs => [
    {:title => ...},
    action_name == "resolve" ? { .. } : nil
  ].compact
}
```

Maybe there's a better technique, it's too early in the morning for me right now :confused: